apps/topcryptoprices/top_crypto_prices.star

```diff
@@ -0,0 +1,331 @@
+"""
+Applet: Top Crypto Prices
+Summary: Top Crypto USD Prices
+Description: This app shows the latest prices (USD) and price changes for the top cryptocurrencies. It is a very simple app, but does exactly what I need and what I bought the Tidbyt for in the first place. Prices are updated through the free Coingecko API at a maximum rate of once per minute.
+Author: playak
+
+Version: 1.1
+Date: 2022-11-09
+Comments: Written to display the price info of top cryptocurrencies on a Tidbyt screen. We manually exclude stable coins and wrapped coins (so we only need one API call).
+Copyright: © 2022 Jeroen Houttuin, Playak - jeroen@playak.com - https://playak.com
+"""
+
+# CONFIG
+TTL = 60  # time to live for the API cache. Be nice to the free Coingecko API provider!
```

Comment:
This is probably good enough, but should mention that there are other apps using the free API as well, so they'll all be sharing the same rate limit (although I'm not sure how Coingecko actually implements that, if at all). If we run into problems we can move this app over to Tidbyt's paid API key.

Reply:
Looks like you got it. I'll merge this and try to get it published later today.